src/app/register/page.tsx

```diff
@@ -0,0 +1,60 @@
+"use client";
+import React, { useState } from 'react';
+
+const Login: React.FC = () => {
+    const [email, setEmail] = useState('');
+    const [password, setPassword] = useState('');
+
+    const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+        event.preventDefault();
+        console.log(email, password);
+    };
+
+    return (
+        <div className="flex items-center justify-center min-h-screen p-4">
+            <form onSubmit={handleSubmit} className="flex flex-col items-center justify-center min-h-screen w-full sm:max-w-md p-8 space-y-6 rounded-lg">
+
+                <div className="w-20 flex flex-col mx-auto justify-center">
+                <img src="/logo.png" alt="Logo" className="mx-auto w-15 h-15 -mt-7" />
+
+                    <h1 className="mx-auto font-bold tracking-wide">OrthoChat</h1>
+                </div>
+                <h3 className="mx-auto text-center">Create your account now</h3>
+                <input
+                    type="email"
+                    value={email}
+                    onChange={(e) => setEmail(e.target.value)}
+                    placeholder="Email"
+                    className="w-full px-4 py-2 border rounded-md focus:outline-none focus:ring-2 focus:ring-blue-500"
+                />
+                <input
+                    type="password"
+                    value={password}
+                    onChange={(e) => setPassword(e.target.value)}
```

Comment:
The way this onChange function is written causes values for the `password` and `confirm password` input boxes to change together.